### ISSUE TITLE
Steam/TTY7 says offline while network connectionS are online/TTY2

### ISSUE THREAD

**Issue by daniellasalle** [ChimeraOS/chimeraos#310]
Following https://github.com/ChimeraOS/chimeraos/issues/163
I was able to login but I am now having a problem with the network: TTY7/GUI doesn't see the network yet TTY2 does.
See attached pictures.

After having issued both the "rm ~/.local/share/Steam/config/config.vdf" and "steam-runtime --reset" commands and having rebooted, I now see the Steam login but I also have this:
![IMG_20220409_110408](https://user-images.githubusercontent.com/26171838/162579958-bb8bc8e0-8228-4ac2-a0d7-ccfaea523245.jpg)

So logging in via "offline mode" and then looking the at TTY7 I have no network:
![IMG_20220409_110427](https://user-images.githubusercontent.com/26171838/162579960-434ee28b-68c7-4457-b976-e9b3c59ddb01.jpg)

But the TTY7 settings also suggests that I am connected to wifi:
![IMG_20220409_110535](https://user-images.githubusercontent.com/26171838/162579961-01187134-c71e-4eaa-a17e-e792c429899c.jpg)

Also TTY2 agrees that the Internet works:
![IMG_20220409_110649](https://user-images.githubusercontent.com/26171838/162579962-f4875865-3325-408d-bc67-97069c7dd755.jpg)

BONUS: I have went ahead and plugged a RJ45 in parallel (of all of this action going) to my wifi connection and even though I also get a 2nd IP on enp2s0 that TTY7 remains the same.

PS: Has an extended troubleshooting bonus I was also getting this problem before I faced the "big blue screen of nope" so maybe they are related?

**Comment by daniellasalle** [comment]
By "firewall issue" you mean the FW of ChimeraOS ofc?

27 May 2022, 11:24 by ***@***.***:

**Comment by Zeitwaechter** [comment]
Ah, yeah, I deleted the message, but of course, there is the email notification stuff.. welp.

Well, I did mean the firewall/blacklist of your router/pihole, but I also did mean another location before even getting to the login screen.<br/>
So that's why I deleted my post, to not hijack your issue/the topic - sorry :)

**Comment by daniellasalle** [comment]
Yeah exactly: email notifs :)

Well thanks for those clarifications but to me it would seem very hard(read; impossible?) that my homemade router did that because there is no such feature.

Also, this problem always happening when the blue screen of nope happened before hints me that the problem is located in the ChimeraOS image rather than in my infrastructure.

But to be 100% sure there's got to be a log somewhere that I could check (in ChimeraOS) which would point to those router denials.

I would... hope?

Regardless, thanks for your participation in this thread and glad(kinda) to know that I am not the only person having an issue.

I guess my case in point is that no matter how good and how much work there was done on this project that it's missing a ishton more to be onpoint.

Of course, I am now ready and willing to make all of this better by actively contributing.

Lastly, I will leave with the closing statement that I strongly believe that ChimeraOS could massively gain from being totally adapted to the Atari VCS and c/should use this opportunity as such.

Anyways, awaiting the next comment/Have a merry weekend :)

27 May 2022, 12:58 by ***@***.***:

**Comment by Zeitwaechter** [comment]
Well.. no problem with that, even though it did not really help you any bit - is what I assume for now.

I would also not say it's about your infrastructure, per say. I just wanted to raise the option that it _might_ be a cause, like it is for my initial boot issue :P

Not to hijack this PR any further, but as you have just stated it, please also checkout #344, which I have just created, related to the ongoing/still happening screen of nope/deep blue ocean/nothingness/whateveryouwannacallit issue.

Aaaand I wish you a nice evening as well! :+1:

**Comment by alkazar** [comment]
Is this still an issue in the latest release?

**Comment by daniellasalle** [comment]
Thanks for your follow-up!

It's actually worst now. See attached.

REPLAY STEPS:
I launched install wizard and got the usual no wired connection error:
![IMG_20220810_202623_983](https://user-images.githubusercontent.com/26171838/184045735-5652b9f2-d48d-4a4f-bf76-afed4c008ae9.jpg)

On the previous version I would enter WPA key and was able to move on with the install, like this:
![IMG_20220810_202637_784](https://user-images.githubusercontent.com/26171838/184045783-d9fb9f1a-b148-4c17-986e-0a68c64227bc.jpg)

But now when I validate said WPA key, it says that it's connecting but it's not:
![IMG_20220810_202701_180](https://user-images.githubusercontent.com/26171838/184045828-8042fc1c-49df-4b3a-88dc-cac3cb139907.jpg)

And after about 60 secs it comes back with "Could not activate connection: Activation failed: IP configuration could not be reserved (no available address, timeout, etc.)":
![IMG_20220810_202734_935](https://user-images.githubusercontent.com/26171838/184045964-dbf98566-6e5b-4f6f-b39e-eaf1b26f4b46.jpg)

Acknowledging this activation connection error via ENTER now re-frames as this:
![IMG_20220810_202806_057](https://user-images.githubusercontent.com/26171838/184046059-f525d5d8-517b-4094-a93d-62343b5defcc.jpg)

And pressing ESC totally breaks it:
![IMG_20220810_202833_613](https://user-images.githubusercontent.com/26171838/184046117-b78accb9-8fb4-472c-8c46-5ccc444cdc81.jpg)

**Comment by Samsagax** [comment]
Is this issue relevant on latest versions? Be aware that both the installer and the image has been updated. The no IP problem is most likely the access point.

**Comment by daniellasalle** [comment]
Thanks for your follow-up! However your reply made me confused.

When you say "on latest versions": I am using the download from chimeraos.org and it still dates back to June 2022 (chimeraos-2022.06.02-x86_64.iso). I am assuming this isn't the one that you are referring to?

When you say "The no IP problem is most likely the access point": can you explain your technical logic when this is a message that comes from the software? Also I need to say that this AP services at least half a dozen mixed wireless clients at all time all of them which work perfectly.

Please advise.


27 Dec 2022, 07:03 by ***@***.***:

**Comment by Zeitwaechter** [comment]
I can also pitch in again, but the first part won't be that helpful at all.

When installing there are times when my normal internet connection does just not work out, then again it's just fine.

Having installed it again in the last few days, this happens on initial check for updates, after a fresh installation. 
Also when I was switching from stable to beta channel.
Otherwise update run through just fine.

I always work around this issue by tethering internet via smartphone/USB, as it does work out. 
Connection via mobile is very slow for me as I tunnel almost everything via it through a VPN.
But.. this works. So.. it's maybe less an issue via ChimeraOS and more one in regardeds to the Firewall/DNS of my Router.

As DNS I use an pi-hole, so this might filter out some of the "bad" URLs from Steam.

Specifically my Steam Deck also does suffer from initial connection issues, every now and then, after Standby.
But.. this does not happen when I'm using other's WLAN.

All in all I therefore think your issues and mine are just Router/Firewall/DNS related and not part of an issue directly related to ChimeraOS.

---

tl;d(n)r - This leads me to a question:

@daniellasalle, did you ever try to setup your System via LAN, instead of WLAN? 
Did/does the behaviour differ? Maybe there is something related to the setup/settings of your router?
If so, nobody can really help you but yourself :(
Did you flash any custom OS on it? Like, e.g., ipfire, dd-wrt, or something else?
That in itself is only benefitial, for sure, but maybe there went something "wrong" as to it not allowing new hosts in and ignoring your MAC filtering for whitelisted addresses, or some other related stuff?

And yes, I still ask these "stupid" questions, even if you have already a dozen of WLAN clients connected all the time :+1:  

I'm just using them to try to pin the issue down more closely. 

You can also just try to setup the WLAN connection afterwards, after having tried and installed everything via LAN. 
When using the `chimera-session desktop` command you can just tinker around with the network settings there again.

Even if there was anything that would be related to the ChimeaOS setup itself, this might be just the workaround for you?

Cheers and let me know if this could help somehow :)

**Comment by daniellasalle** [comment]
Hello again and thanks for all of that detailed technical information.

Concerning the "not so useful" first part, here is what I have to say:

In terms of Inet connection and infrastructure stability and reliability, I have a fiber link. So I am required to rule out this as being part of the problem.

When you say "Having installed it again in the last  few days", are you referring to the ChimeraOS.iso or your Inet link? Logic tends me to push towards the ISO. However I would like to think that installing it multiple times most possibly came out directly off the git commits (Sorry but I do not stay informed with the unstable branch and only/always download the .ISO directly off the .org.

In terms of tunneling everything via VPN and it doesn't always working out: I too am using a VPN and I must admit that my fiber link takes a major dump in terms of performance. My point is, if your Inet link is basic speeds by default that most probably the software isn't to blame. Yes, I really like ChimeraOS so much that I will defend it until I no longer can :)

I also do not understand why your router/modem (are they the same physical box to begin?) would be the device giving you grief if other devices are (mostly) working well all of the time. However I will hint you that router/AP capacity (CPU+memory+supported specs/etc) will need to reviewed from time to time. At first I didn't believe in those "gaming AP" as I thought it was just marketing but time proved me wrong. I will also assume that you already know all of this.

Perhaps the most important point of that first "not so useful" part is the Steam Deck. I too have one and never have had this problem with it. I have used it for over 80 hours and have tried it on 2 APs and never had this issue. Maybe this problem really isn't the SD but rather that it manifests to it. Food for thought.

Now moving on to the most important part, the questions:

1. Setting up my VCS via LAN was one of the first thing that I did. And the behaviour was the same.
2. My AP(s) are fully "vendor original" meaning they run the latest firmware of the vendor themselves and nothing that is unsupported.

The problem I have with everything that we've talked is that per the pictures I have provided we can clearly see that the software has a hard time managing the "echos"(for a lack of a better term) of the device. I am pretty sure that something is happening during the install shield that doesn't parse properly and that ends up messing up the user interface and then it dies.

But moving forward: when is the next stable-official .ISO dropping on chimeraos.org?

Bonus mission: should I git clone and then build a .ISO from latest branch (or whatever?), transfer it to a USB drive and try again? Per previously mentioned I am wearing the end user googles here therefore I require some guidance given that we wish to go that route.

Cheers =D


6 Jan 2023, 03:31 by ***@***.***:
